boot.php

```diff
@@ -21,16 +21,15 @@
 
 use Friendica\App;
 use Friendica\Core\Addon;
-use Friendica\Core\System;
-use Friendica\Core\Cache;
 use Friendica\Core\Config;
-use Friendida\Core\L10n;
+use Friendica\Core\L10n;
 use Friendica\Core\PConfig;
+use Friendica\Core\System;
 use Friendica\Core\Worker;
 use Friendica\Database\DBM;
-use Friendica\Model\Contact;
 use Friendica\Database\DBStructure;
-use Friendica\Module\Login;
+use Friendica\Model\Contact;
+use Friendica\Util\DateTimeFormat;
```

Comment:
Just curious, is there a reasoning behind this constantly re-sorting of the `use` statements?

Reply:
I have an automatic tool that checks for class usage inside a file and suggest a use list alphabetically sorted. Not everyone use this tool, and sometimes people add uses that aren't alphabetically sorted. If I'm subsequently editing the file and using my IDE tool, it sorts the use statements back.